05.wc/wc.rb

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require 'optparse'
+require 'pathname'
+require 'find'
+
+def check_opt
+  opt = OptionParser.new
+  options = {}
+  opt.on('-l') { |lines| options[:l] = lines }
+  opt.on('-w') { |words| options[:w] = words }
+  opt.on('-c') { |byte| options[:c] = byte }
+  opt.parse!(ARGV)
+  options = { c: true, l: true, w: true } if options.empty?
+  options
```

Comment:
```suggestion
  options.empty? ? { c: true, l: true, w: true } : options
```

なるべく再代入をせずに済むロジックにしましょう
https://bootcamp.fjord.jp/pages/avoid-re-assign

Reply:
`return { c: true, l: true, w: true } if options.empty?`として、メソッド内のoptionsに再代入しない形に変更いたしました。